docs/nakama-download.md

```diff
@@ -0,0 +1,41 @@
+# Download
+
+All Nakama versions are available for download from GitHub Release page. In addition, pre-build Docker images are available on Docker Hub.
```

Comment:
"for download from GitHub Release page" -> "for download from the GitHub Release page"

What does "pre-build" mean with the Docker images?